client/blocks/post-likes/index.jsx

```diff
@@ -24,7 +24,7 @@ class PostLikes extends PureComponent {
 	renderLike = ( like ) => {
 		const { showDisplayNames } = this.props;
 
-		const likeUrl = like.site_ID && like.site_visible ? '/read/blogs/' + like.site_ID : null;
+		const likeUrl = like.login ? '/read/users/' + like.login : null;
```

Comment:
```suggestion
		const likeUrl = like.login ? getUserProfileUrl( like.login ) : null;
```

Reply:
Oh, I forgot about that function. Good call, thanks!